panel/webserver_configuration.md

```diff
@@ -50,3 +50,48 @@ sudo ln -s /etc/apache2/sites-available/pterodactyl.conf /etc/apache2/sites-enab
 sudo a2enmod rewrite
 systemctl restart apache2
 ```
+
+## Caddy
+Caddy is a webserver with automatic https, this may come handy when you don't want to remake your ssl certificates every 90 days.
+::: danger
+Caddy is incompatible with other webservers, if you plan on running other websites on your panel you will have to add them to caddy too.
+:::
+/etc/caddy/Caddyfile:
+``` json
+panel.yourdomain.com {
+    # Set the directory root
+    root /var/www/pterodactyl/public
+    # Let caddy know where fastcgi is for php, edit "7.2" to your php-fpm version
+    fastcgi / /run/php/php7.2-fpm.sock php
+    # rewrite all requests to index.php, this is required for how pterodactyl works
+    rewrite {
+        to {path} {path}/ /index.php
+}
+```
+to turn it on:
+``` bash
+caddy -conf /etc/caddy/Caddyfile
+```
+To run caddy in the background use SystemD
+``` bash
+cat <<EOF > /etc/systemd/system/caddy@.service
+[Unit]
+Description=Caddy HTTP/2 web server %I
+Documentation=https://caddyserver.com/docs
+After=network.target
+
+[Service]
+User=%i
+Environment=STNORESTART=yes
+ExecStart=/usr/local/bin/caddy -agree=true -conf=/etc/caddy/Caddyfile
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+EOF
+```
+reload the systemd daemon: `systemctl daemon-reload`
```

Comment:
Please clean up these instructions, they do not match the rest of the documentation, and a majority of them are unnecessary. We do not teach people how to use dependencies in this documentation, so things like how to restart the webserver or start it shouldn't be included unless they are relevant to the process itself.